.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
-version: 2.1
-
 exec: &exec
   name: build-tools/nerves-system-br
+  version: 1.21.0
+  elixir: 1.13.4-otp-25
   version: 1.20.5
   elixir: 1.13.4-otp-25
```

Comment:
Looks like you accidentally added the options again rather than overwrite

Reply:
It's the script. You looked too quickly. :)

Reply:
🏎️ 💨